.github/workflows/main.yml

```diff
@@ -0,0 +1,121 @@
+name: Playwright Tests
+on:
+  pull_request:
+    branches:
+      - develop
+      - master
+      - release-*
+jobs:
+  
+  linux:
+    name: ${{ matrix.os }} (${{ matrix.browser }})
+    strategy:
+      fail-fast: false
+      matrix:
+        browser: [chromium, firefox]
+        os: [ubuntu-18.04]
+        nodeversion: ['16']
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        cache: npm
+        node-version: ${{ matrix.nodeversion }}
+    - name: Cache node modules
+      uses: actions/cache@v2
+      env:
+        cache-name: cache-node-modules
+      with:
+        # npm cache files are stored in `~/.npm` on Linux/macOS
+        path: ~/.npm
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-build-${{ env.cache-name }}-
+          ${{ runner.os }}-build-
+          ${{ runner.os }}-
+    - name: Setup custom host for sdkapp.example.com
+      run: echo "127.0.0.1 sdkapp.example.com auth.example.com api.example.com openam.example.com" | sudo tee -a /etc/hosts
```

Comment:
I don't believe any E2E tests use `openam.example.com`. This domain can be found in the codebase, but they are all unit or integration tests, which neither use a real server.